diff --git a/src/pretraining/wandb/latest-run b/src/pretraining/wandb/latest-run
index e930720..caf21b8 120000
--- a/src/pretraining/wandb/latest-run
+++ b/src/pretraining/wandb/latest-run
@@ -1 +1 @@
-run-20220930_125524-31oqthjt
\ No newline at end of file
+run-20221004_114259-1dtinakr
\ No newline at end of file
diff --git a/src/pretraining/wav2vec2_pretraining.py b/src/pretraining/wav2vec2_pretraining.py
index f5c813e..c885b25 100644
--- a/src/pretraining/wav2vec2_pretraining.py
+++ b/src/pretraining/wav2vec2_pretraining.py
@@ -22,7 +22,7 @@ def parse_args():
     parser.add_argument(
         "--dataset",
         type=str,
-        default=None,
+        default="~/speech-processing/data/processed/libri_vectorized",
         help="Vectorized dataset directory",
     )
     parser.add_argument(
